app/templates/components/orange-bar-data-explorer.hbs

```diff
@@ -0,0 +1 @@
+<li>Orange bar for the data explorer page</li>
```

Comment:
<img width="258" alt="image" src="https://user-images.githubusercontent.com/5004319/121229350-abc2d000-c842-11eb-9f64-25e63545b25d.png">

I like to check for these in general.